hw/hw8/processor.py

```diff
@@ -0,0 +1,34 @@
+def open_file():
+    path = open('phoneBook.txt', 'a')
+    my_file = open(path, 'r')
```

Comment:
Тут что-то странное. Открыт файл, а потом он еще раз открывается, только в качестве пути указан другой файл. И режимы чтения разные. Ничего не понимаю. Вы же только читаете , а не записываете. Зачем вам режим дозаписи? И про кодировку не забывайте
```suggestion
    my_file = open('phoneBook.txt', encoding='utf-8')
```